src/store/staking-applications/slice.ts

```diff
@@ -1,326 +0,0 @@
-import {
-  AnyAction,
-  createSlice,
-  isAnyOf,
-  PayloadAction,
-} from "@reduxjs/toolkit"
-import { BigNumber } from "ethers"
-import { featureFlags } from "../../constants"
-import {
-  StakingProviderAppInfo,
-  AuthorizationParameters,
-} from "../../threshold-ts/applications"
-import { AddressZero, MAX_UINT64 } from "../../threshold-ts/utils"
```

Comment:
I think that we should also remove everything related to staking from `threshold-ts` lib. In the perfect world it would be best to have `threshold-ts` as a separate lib (in separate repo) and use it in both project.

The `threshold-ts` lib in this project is outdated already due to some changes that were made in [Taco PR](https://github.com/threshold-network/token-dashboard/pull/625) in the `token-dashboard`, so we won't be able to keep those two the same. I would say that we should remove all unnecessary things from it in this PR, and just leave the things that are relatd to TBTC.